### DISCUSSION TITLE
Searchable dropdown for ediable column? Similar to searchable dropdown in forms.

### DISCUSSION THREAD

**Discussion by shivaRamdeen** [filamentphp/filament#12774] [Help]
### Package

Table builder

### Package Version

v3.0.0

### How can we help you?

Has anyone come across any good searchable dropdown plugins for table builder? The current dropdown loads all records which is over 7k in my case. 

Unfortinately I'm coming from a vanilla laravel + vuejs background so imlementing a custom solution with all of the new tech that is used by Filament is very slow.

Any suggestions are highly appreciated.

**Comment by leandrocfe** [comment]
Custom column I think
https://filamentphp.com/docs/3.x/tables/columns/custom
